R/compute_centiloid.R

```diff
@@ -0,0 +1,176 @@
+#' Compute Centiloid Value
+#'
+#' Computes centiloid values based on amyloid PET tracer, SUVR value and pipeline,
+#' and reference region.
+#' Also allows for custom formula parameters.
+#'
+#' @param tracer Amyloid PET tracer from ag_neuro dataset
+#'
+#'   A character string is expected.
+#'   When custom parameters are not provided, accepted values are:
+#'   '18F-Florbetapir', '18F-Florbetaben'
+#'   If custom slope and intercept are provided, any string is accepted.
+#'
+#' @param pipeline SUVR pipeline from nv_neuro dataset
+#'
+#'   A character string is expected.
+#'   When custom parameters are not provided, accepted values are:
+#'   'AVID FBP SUVR PIPELINE', 'BERKELEY FBP SUVR PIPELINE', 'BERKELEY FBB SUVR PIPELINE'
+#'   If custom slope and intercept are provided, any string is accepted.
+#'
+#' @param ref_region Reference region from suppnv_neuro dataset
+#'
+#'   A character string is expected.
+#'   When custom parameters are not provided, accepted values are:
```

Comment:
```suggestion
```suggestion
#'  See Details section for accepted values in combination with `tracer` and `pipeline`.
```
```